src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap

```diff
@@ -143,6 +143,26 @@ Object {
 }
 `;
 
+exports[`Canonical Media Asset Page I can see related content: "المايوه الشرعي" يثير جدلا في الساحل الشمالي في مصر 1`] = `"/arabic/tv-and-radio-53492142"`;
```

Comment:
I did a PR to remove these recently, do you have any idea why they keep appearing a re-appearing (I may have been wrong to remove them but my int test run pointed out obsolete snapshots.

Reply:
Yeh I was confused about this. I thought I fixed the issue of int tests not failing CI when they don't pass. I'm going to look into this again soon.